client/src/context/useAuthContext.tsx

```diff
@@ -2,20 +2,15 @@ import { useState, useContext, createContext, FunctionComponent, useEffect, useC
 import { useHistory } from 'react-router-dom';
 import { AuthApiData, AuthApiDataSuccess } from '../interface/AuthApiData';
 import { User } from '../interface/User';
-import loginWithCookies from '../helpers/APICalls/loginWithCookies';
-import logoutAPI from '../helpers/APICalls/logout';
+import { loginWithCookies, logout as logoutApi } from '../helpers/';
 
 interface IAuthContext {
   loggedInUser: User | null | undefined;
   updateLoginContext: (data: AuthApiDataSuccess) => void;
   logout: () => void;
 }
 
-export const AuthContext = createContext<IAuthContext>({
-  loggedInUser: undefined,
-  updateLoginContext: () => null,
-  logout: () => null,
-});
+export const AuthContext = createContext<IAuthContext>({} as IAuthContext);
```

Comment:
Removing unnecessary data in context as it only needs to hold shape and not necessarily variables/functions